sdk/template-dpg/Azure.ServiceTemplate.Template/src/autorest.md

```diff
@@ -7,10 +7,11 @@ Run `dotnet build /t:GenerateCode` to generate code.
 
 ``` yaml
 input-file:
-- $(this-folder)/swagger/swagger.json
-namespace: Azure.Template.Generated
+- SwaggerFileLink
```

Comment:
Use `<link_to_swagger>` notation to make it clear that this needs to be filled in?  Consider linking to an example here, as there are multiple ways to do it in the yaml format.

Reply:
It is placeholder, and it will be replaced automatically to the input files. customer may not see this after the package is generated.